src/main/java/app/attestation/server/AttestationServer.java

```diff
@@ -89,6 +89,12 @@
     private static final int BUSY_TIMEOUT = 10 * 1000;
     private static final int QR_CODE_PIXEL_SIZE = 300;
     private static final long SESSION_LENGTH = 48 * 60 * 60 * 1000;
+    private static final int HISTORY_PER_PAGE = 20;
+
+    private static final String BASEURL = "staging.attestation.app";
```

Comment:
Omit this for now. It doesn't do anything beyond being checked in Auditor anyway. Makes sense to split it up like this and use a different one for debug builds of Auditor but it needs to be kept in sync and should be done separately from this PR.